app/components/Calendar/messages.js

```diff
@@ -0,0 +1,28 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  minPeriodText: {
+    id: 'Calendar.info.minPeriodText',
+    defaultMessage: 'Reservation must be at least {duration}h long',
+  },
+  maxPeriodText: {
+    id: 'Calendar.info.maxPeriodText',
+    defaultMessage: 'Reservation must be shorter than {duration}h',
+  },
+  today: {
+    id: 'Calendar.info.today',
+    defaultMessage: 'Today',
+  },
+  selectedDateLabel: {
+    id: 'Calendar.selectedDateLabel',
+    defaultMessage: 'Valittu aika:',
+  },
```

Comment:
I might be a little too picky, but would it be a good idea to keep defaultMessage language consistent?

Reply:
Good point. I changed them to Finnish because that's used elsewhere in the app.